users/tests/test_views.py

```diff
@@ -272,3 +277,67 @@ def test_registered_user_with_wrong_activation_link_will_not_activate_account(se
             response = self.client.get(path=url)
             self.assertEqual(response.status_code, 200)
             self.assertEqual(CustomUser.objects.get(email=self.user.email).is_active, False)
+
+
+class NotificationsTests(TestCase):
+    def setUp(self):
+        self.admin = AdminUserFactory()
+        self.employee = UserFactory()
+        self.manager = ManagerUserFactory()
+        self.project = ProjectFactory()
+        self.project.managers.add(self.admin)
+        self.project.managers.add(self.manager)
+        self.project.members.add(self.employee)
+        self.url = reverse("custom-users-notifications")
+
+    def _check_response(self, response, status_code, contains):
+        self.assertTemplateUsed(response, NotificationUserListView.template_name)
+        self.assertEqual(response.status_code, status_code)
+        for element in contains:
+            self.assertContains(response, element)
+
+    def test_manager_should_be_able_get_notification_view(self):
```

Comment:
You should also add this check to `AccessPermissionsTestCase`.

Reply:
You're right. I've deleted them from `test_views` an`d moved to `test_access_permissions`